src/actions/websocket.ts

```diff
@@ -611,11 +616,8 @@ function handleChannelDeletedEvent(msg: WebSocketMessage) {
                 }
                 EventEmitter.emit(General.DEFAULT_CHANNEL, '');
             }
-
-            dispatch({type: ChannelTypes.RECEIVED_CHANNEL_DELETED, data: {id: msg.data.channel_id, team_id: msg.data.team_id, viewArchivedChannels}});
-
-            dispatch(fetchMyChannelsAndMembers(currentTeamId));
```

Comment:
Why did you remove this dispatch? Is not needed?

Reply:
We already have the channels at this point, so I don't see a need to `re-fetch` them, if you think otherwise, let me know why and I can change it back.